phpunit/blocks/render-query-test.php

```diff
@@ -240,40 +203,4 @@ public function test_rendering_nested_queries_with_enhanced_pagination_auto_disa
 		$router_config = wp_interactivity_config( 'core/router' );
 		$this->assertTrue( $router_config['clientNavigationDisabled'] );
 	}
-
-	/**
-	 * Tests that the `core/query` block sets the option
-	 * `clientNavigationDisabled` to `true` in the `core/router` store config
-	 * when a plugin that does not define clientNavigation is found inside.
-	 */
```

Comment:
If we change Posty Content for `test/plugin-block`, this test is almost identical to [this other one](https://github.com/WordPress/gutenberg/blob/0a36b7881fcb280f28d3339e6d3ccc1167b8423f/phpunit/blocks/render-query-test.php#L115). Do you think it is worth keeping both?